LiteCore/tests/LazyVectorAPITest.cc

```diff
@@ -145,19 +145,19 @@ class LazyVectorAPITest : public C4Test {
         } catch ( [[maybe_unused]] std::exception& e ) { return nullptr; }
     }
 
-    void checkQueryReturnsWords(C4Query* query, const std::vector<string>& expectedWords) const {
-        auto e = REQUIRED(c4query_run(query, _encodedTarget, ERROR_INFO()));
-        REQUIRE(c4queryenum_getRowCount(e, ERROR_INFO()) == expectedWords.size());
-        for ( const auto& expectedWord : expectedWords ) {
-            REQUIRE(c4queryenum_next(e, ERROR_INFO()));
-            FLArrayIterator columns  = e->columns;
-            slice           word     = Value(FLArrayIterator_GetValueAt(&columns, 0)).asString();
-            float           distance = Value(FLArrayIterator_GetValueAt(&columns, 1)).asFloat();
-            CHECK(word == slice(expectedWord));
-        }
-        CHECK(!c4queryenum_next(e, ERROR_INFO()));
-        c4queryenum_release(e);
-    }
+    //    void checkQueryReturnsWords(C4Query* query, const std::vector<string>& expectedWords) const {
```

Comment:
This piece comes from @callumbirks 's PR, (#2052/6b3045578111c). 
Already communicated with him and he will use the function in a separate PR.
Because my xcode won't let me pass for unused variable, "distance", and I found the entire function is not used so, just commented it out.

Reply:
I saw his new PR is coming. It will be taken care of after both been merged.